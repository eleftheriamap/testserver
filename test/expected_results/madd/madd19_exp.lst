
test/exp/test/gen/madd/madd19_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	114b5e94 	add	w20, w20, #0x2d7, lsl #12
   4:	11177000 	add	w0, w0, #0x5dc
   8:	d1783084 	sub	x4, x4, #0xe0c, lsl #12
   c:	1b001281 	madd	w1, w20, w0, w4
  10:	8a000000 	and	x0, x0, x0
