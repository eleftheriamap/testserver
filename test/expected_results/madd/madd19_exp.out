Registers:
X00    = 00000000000005dc
X01    = 00000000095b8000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = ffffffffff1f4000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 00000000002d7000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x114b5e94
0x00000004 : 0x11177000
0x00000008 : 0xd1783084
0x0000000c : 0x1b001281
0x00000010 : 0x8a000000
